egs2/chime7_task1/diar_asr1/run.sh

```diff
@@ -65,6 +66,20 @@ if [ -z "$gss_asr_stage" ]; then
 fi
 
 
+if [ "${decode_only}" == "eval" ]; then
+  diar_inf_dset="eval"
+fi
+
+
+if [ -n "${download_baseline_diarization}" ]; then
+  # TODO
+  log "Using organizer-provided JSON manifests from the baseline diarization system."
+  #git clone
+  #mv
+  stage=4
```

Comment:
Just want to double check if we can skip stage 3 if we have this. How can we download baseline diarization results?